srcpkgs/qutebrowser/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'qutebrowser'
 pkgname=qutebrowser
-version=1.14.0
+version=1.14.1
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools asciidoc"
 depends="python3-PyQt5-quick python3-Jinja2 python3-Pygments python3-pyPEG2
  python3-yaml python3-attrs python3-PyQt5-opengl python3-PyQt5-sql
- qt5-plugin-sqlite"
+ qt5-plugin-sqlite python3-setuptools"
```

Comment:
Wonder why it started being required... Did it fail at install time or runtime as well?

Reply:
Ah, it uses `pkg_resources` in a bunch of places. Thanks for spotting!

Reply:
QB was about the first thing I installed on a new void. It xi'd just fine, but it failed at runtime.

Reply:
That makes sense ;)